1-exercises/A-reading-list/script.js

```diff
@@ -1,5 +1,29 @@
+let content = document.querySelector('#content')
+let listOfBooksUlEl= document.createElement('UL')
+
 function readingList(books) {
-  // Write your code here...
+  for(let i=0; i<books.length; i++){
+
+    let newParEl = document.createElement('p');
+    newParEl.innerHTML=books[i].title + ' by '+ books[i].author;
+
+    let bookCoverEl = document.createElement('img')
+    bookCoverEl.src=books[i].bookCoverImage;
+    bookCoverEl.width='200'
+
+    let listOfBooksEl= document.createElement('LI')
+    if (books[i].alreadyRead){
+      listOfBooksEl.className= 'green'
+    }
+    else{
+      listOfBooksEl.className = 'red'
+    }
+
+    listOfBooksEl.appendChild(newParEl)
+    listOfBooksEl.appendChild(bookCoverEl)
+    listOfBooksUlEl.appendChild(listOfBooksEl)
+    content.appendChild(listOfBooksUlEl)
```

Comment:
This line only needs to be called once as editing the children of the UL element will show the changes in the UI if the UL element is a child of the contentElement. May be worth moving this line outside the for loop.